### PR TITLE
Feature: Google maps API

### PR DESCRIPTION
**Pull request by kamilsdz**
Code from this PR:
- looking for places nearby (longitude, latitude)
- is providing detailed information about place from (longitude, latitude)
- is adding page 'find_nearby' for geolocation feature
- tries to download POI data during its import

Still to do:
*  add new attributes to Workshop model (my suggestion: 
      rating,
      place_id,
      name,
      adress,
      phone )
* add geolocation fields to user model
* handle request with geolocation

### CODE REVIEW

File: lib/google_api.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  def objects_to_array(latitude, longitude, type)
+    places = []
+    find_objects_nearby(latitude, longitude, type)['results'].each do |object|
+      places << get_place_details(object['place_id'])['result']
+    end
+    places.compact
+  end
+
+  def secrets
+    ENV['GOOGLE_API_KEY']
```
Comment (fab-ian) [inline lib/google_api.rb]: Manage environment variable in project and on server

Comment (kamilsdz) [reply lib/google_api.rb]: Off course, I'll do it. We should also decide - do we want to get data from the API when importing POIs or not?
If not - I'll restore the previous version of PoiImporter service. @fab-ian  @nikos83

Comment (fab-ian) [reply lib/google_api.rb]: @kamilsdz Frankly speaking I'm not really understand the new idea, but I didn't spend much time to analyse this solution. Maybe first let's us build the 'nearest objects' block and then we consider what's next.

File: .env_sample [hunk 2]
```diff
@@ -0,0 +1 @@
+GOOGLE_API_KEY="secret api key"
```
Comment (fab-ian) [inline .env_sample]: There are more keys :) We need all of them.

Comment (fab-ian) [reply .env_sample]: Ok, if you fix merge conflict all will be ok.

Comment (kamilsdz) [reply .env_sample]: Yep, I started this branch before you did dotenv.

**Pull request merged**